app/views/questions/show.html.erb

```diff
@@ -12,6 +12,9 @@
         <br>
         <div id="question-body" >
```

Comment:
I like that you labeled this div. Makes the html read so much better as it gets bigger.